### ISSUE TITLE
on change of files - maestro debugging fails - need restart ide

### ISSUE THREAD

**Issue by xsoft7** [mbolotov/intellij-playwright#14]
it's happening since version 1.2

each time there's a change in the test data, the debug session button becomes white (instead of red) and it fails to catch the change

lets say i had a param:

let x=3

and then i changed to:

let x = 4

when i press debug, it will skip the change as if there wasnt a debug point there.
 see image below of 'white debugging point'
![image](https://user-images.githubusercontent.com/38626413/194321766-1e354192-286a-4c84-ade6-97ea190230e0.png)

only after restrat of the IDE it works.

**Comment by mbolotov** [comment]
Hello, 
I'm currently unable to reproduce the problem and not sure if I understand it correctly. 
where is that `x` variable coming from? can you add a full screenshot?
it will be perfect if you provide a video recording of the problem.

**Comment by xsoft7** [comment]
i've uploaded a video.
https://youtu.be/5nDiRc9QkRQ

as you can see,1) i've started the video with a problem. (debug is muted)

for the debug points to work -  i have to restart the IDE.


2) after retart the ide- debug works.

3) i've uncommented the line -  debug works

4) i;ve commented the line -debug doesnt work

5) restart the ide - debug works again

**Comment by xsoft7** [comment]
any updates on the matter? - just received update for webstorm - to  2022.2.3 . the problem persists.

**Comment by mbolotov** [comment]
Okay, I see how you do reproduce the problem. Unfortunately, I'm unable to do so on my side.
I will soon look into the problem deeper. Meanwhile, you can try this bundle:
[Maestro-1.2-EAP2.zip](https://github.com/mbolotov/intellij-playwright/files/9752869/Maestro-1.2-EAP2.zip)
It should not mute the breakpoints, but (for me) it has another problem - when terminating the Playwright process by IDE, it works as just 'Resume' action instead of termination. And it will so stop on any further breakpoint or run until the end normally. 
Please see if you notice the same behavior.

**Comment by xsoft7** [comment]
yes can confirm that:

1) it solved the original issue (no need to restart the IDE after changes) -  what was the issue?

2) it added new issue - stopping in a debug mode- resumes the session, instead of stopping it.


so, to mitigate the issues , is there a version of webstorm to install and use ?

**Comment by mbolotov** [comment]
while I'm investigating the issues, you can downgrade to 1.2.1 version:
[Maestro-1.1.2.zip](https://github.com/mbolotov/intellij-playwright/files/9753998/Maestro-1.1.2.zip)

**Comment by mbolotov** [comment]
Hi @xsoft7!
While I'm still investigating the problems, I think I found a workaround to make all the functionality working smoothly. 
You can use the [Maestro-1.2-EAP2.zip](https://github.com/mbolotov/intellij-playwright/files/9752869/Maestro-1.2-EAP2.zip) and set the `disable.winp` registry key to `true`. Please see [this SO answer](https://stackoverflow.com/a/28419108/13109074) if you need help on how to manage the IDE registry.
